bookkeeper-http/http-server/src/main/java/org/apache/bookkeeper/http/HttpRouter.java

```diff
@@ -51,6 +51,8 @@
     public static final String BOOKIE_STATE_READONLY        = "/api/v1/bookie/state/readonly";
     public static final String BOOKIE_IS_READY              = "/api/v1/bookie/is_ready";
     public static final String BOOKIE_INFO                  = "/api/v1/bookie/info";
+    public static final String SUSPEND_COMPACTION           = "/api/v1/bookie/compaction/suspend";
```

Comment:
we are using "/gc" (just above) and "/compaction" to control different features of the same functionality. We need to make it less confusing and unify around the same naming.
Maybe deprecate "/api/v1/bookie/gc" / "/api/v1/bookie/gc_details" and do "/api/v1/bookie/compaction/force_major|force_minor" / "/api/v1/bookie/compaction/details" instead?

Reply:
Bookkeeper GC contains two parts, 1. GC ledgers and entry log. 2. compaction entry log files. 
I don't think  we should combine the two apis together.
`/gc` is to trigger 1 and 2, while the new api is to control part 2.

Reply:
+1 for @dlg99. I prefer to use `gc` instead of `compaction` endpoint. It won't introduce break changes.